common/process/inc/com/centreon/common/process/process_args.hh

```diff
@@ -23,28 +23,23 @@ namespace com::centreon::common {
 
 /**
  * @brief The goal of this class is two store arguments in two formats
- * a vector<string> for windows boost process launcher
- * a vector<const char*> for spawnp
+ * a vector<string> for windows boost process launcher, this vector does not
+ * contain exe
+ * a vector<const char*> for spawnp, this vector contains exe and a
+ * nullptr at the end
  *
  */
 class process_args {
   std::string _exe_path;
   std::vector<std::string> _args;
   std::vector<const char*> _c_args;
+  std::unique_ptr<char[]> _buffer;
```

Comment:
What's its purpose?
It is not always initialized and I didn't see where it is used.